### DISCUSSION TITLE
connection yolo to twitch stream

### DISCUSSION THREAD

**Discussion by RNGIllSkillz** [ultralytics/yolov5#6595] [Q&A]
Is it possible to hook up yolo to a stream video? File formats like m3u8 are not supported, is there another way ?

**Comment by glenn-jocher** [comment]
@RNGIllSkillz on the streaming side any RTSP, HTTP, RTMP addresses are supported, as well as YouTube addresses. 

Supported video formats are here, you might try adding your suffix to the list and see if it works:
https://github.com/ultralytics/yolov5/blob/c21da596f3d8db26279c7a7f3c4cfb1a4a8640fc/utils/datasets.py#L34-L38
